bin/rucio

```diff
@@ -1499,7 +1499,11 @@ def list_rse_attributes(args):
     """
     client = get_client(args)
     attributes = client.list_rse_attributes(rse=args.rse)
-    table = [(k + ':', str(v)) for (k, v) in sorted(attributes.items())]
+    table = []
+    for (k, v) in sorted(attributes.items()):
+        if type(v) == float:
+            v = "\'" + str(v) + "\'"
+        table.append((k + ':', v))
```

Comment:
Implemented via alternative method & bypasses the `float` value check & variable conversion but has an extra pair of `' '` as in the output field of the table generated 😥

Reply:
haha, nice. i will try this out to see what it does on our prod instance